content/en/real_user_monitoring/guide/best-practices-tracing-native-ios-android-apps.md

```diff
@@ -0,0 +1,167 @@
+  tag: 'Documentation'
+  text: 'Connect RUM and Traces'
+- link: '/tracing/trace_collection/automatic_instrumentation/dd_libraries/ios'
+  tag: 'Documentation'
+  text: 'iOS Trace SDK'
+- link: '/tracing/trace_collection/automatic_instrumentation/dd_libraries/android/'
+  tag: 'Documentation'
+  text: 'Android Trace SDK'
+- link: '/tracing/trace_collection/custom_instrumentation/ios/otel/'
+  tag: 'Documentation'
+  text: 'OpenTelemetry for iOS'
+- link: '/tracing/trace_collection/custom_instrumentation/android/otel/'
+  tag: 'Documentation'
+  text: 'OpenTelemetry for Android'
+---
+
+# Overview
+
+Datadog's Trace SDK for [iOS][1] and [Android][2] lets you add APM spans to your mobile apps. This guide covers usage, key use cases, and sampling rates, with or without using the Datadog RUM SDK.
+
+Native mobile tracing gives you precise control over what operations are measured by manually instrumenting spans in your iOS or Android app code. This approach works independently of Datadog RUM, but can also be used alongside it for deeper visibility into user experiences, code level performance, and backend interactions. You can also use [OpenTelemetry for iOS][3] or [OpenTelemetry for Android][4] for custom instrumentation.
+
+**Note**: When using the Trace SDK independently (without the RUM SDK), root spans for outgoing network requests do not get automatically created. Therefore, you need to manually start and stop spans on the client side to create frontend-to-backend distributed APM traces.
+
+This guide focuses on native iOS and Android apps. For other mobile platforms, see the following documentation:
+
+- [React Native RUM and APM][8]
+- [Flutter RUM and APM][9]
+- [Kotlin Multiplatform RUM and APM][10]
+- [Unity RUM and APM][11]
+
+For a comprehensive overview of correlating RUM with other telemetry data, see [Correlate RUM with other telemetry data][12].
+
+## Use cases
+
+### Wrap a frontend-to-backend distributed trace under a native span
+
+You can create distributed traces that span from your mobile frontend to your backend services. The RUM SDK provides an automated way to produce client spans for outgoing network requests. Datadog's Trace SDK provides similar capability, but also allows you to wrap one or more frontend-to-backend traces under manually created spans.
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+Use the [manual context propagation][1] to inject trace headers into network requests. Use `setActive()` to link child spans to a parent span ([API reference][2]).
+
+[1]: /tracing/trace_collection/automatic_instrumentation/dd_libraries/ios?tab=swiftpackagemanagerspm#:~:text=(Optional)%20To%20distribute%20traces%20between%20your%20environments%2C%20for%20example%20frontend%20%2D%20backend%2C%20you%20can%20either%20do%20it%20manually%20or%20leverage%20our%20auto%20instrumentation.%20In%20both%20cases%2C%20network%20traces%20are%20sampled%20with%20an%20adjustable%20sampling%20rate.%20A%20sampling%20of%2020%25%20is%20applied%20by%20default
```

Comment:
This link works great now, but I'm wondering if it could introduce a maintenance burden if the text on the linked page ever changes and this link needs to change too in order to keep working.